compress.bash

```diff
@@ -11,19 +11,19 @@
 # Name of the ROM. No Spaces Please.
 # Example: CyanogenMod
 
-name=ColorOS
+name=Ubuntu Touch 6.0.0_r1
```

Comment:
Spaces will break the script. Make the name `UbuntuTouch` 
@regalstreak, remind me tomorrow to come up with a fix for this.